sdk/tests/conformance2/misc/00_test_list.txt

```diff
@@ -1,6 +1,7 @@
 expando-loss-2.html
 getextension-while-pbo-bound-stability.html
 instanceof-test.html
+null-object-behaviour-2.html
```

Comment:
I think technically this should be `>= 2.0.1`.
Given git, I'm not sure we care though?

Reply:
@kenrussell

Reply:
Just for bookkeeping purposes - since we've done this so far - it would be helpful to add "--min-version 2.0.1".